pkg/reconciler/events/storage/storage.go

```diff
@@ -43,7 +43,12 @@ import (
 )
 
 const (
-	resourceGroup = "cloudstoragesources.events.cloud.google.com"
```

Comment:
same comment as other PR. Let's group the reasons together, and let finalizerName and resourceGroup be separate

Reply:
done